src/ci/docker/dist-riscv64-linux/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM debian:10
```

Comment:
All of our images are based off Ubuntu, is there any reason you chose Debian here?

Reply:
This is now changed to Ubuntu 20.04.